cf/src/main/scala/com/timeout/scalacloudformation/Parameter.scala

```diff
@@ -0,0 +1,82 @@
+package com.timeout.scalacloudformation
+import enum.Enum
+
+sealed trait Parameter extends HasLogicalId {
+  def logicalId: String
+  def Type: Parameter.DataType
+  def Description: Option[String]
+  def NoEcho: Option[Boolean]
+}
+
+object Parameter {
+  sealed trait AwsType
+  object AwsType {
+    case object `AWS::EC2::Instance` extends AwsType
```

Comment:
Are you sure about this type? The spec only talks about `AWS::EC2::Instance::Id`, which you added below